.github/workflows/node.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x]
+        node-version: [14.x]
```

Comment:
Let's lose the matrix, so simplify the workflow names

Reply:
Adjusted